README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
+
 **b)** `"1" === 1`
 
+False - 3 ==='s are looking for a 100% match, and string 1 is not exactly the same as number 1.
+Console says: False
+
 **c)** `x == 'x'`
 
+True - Again, because double ='s are looking to see a close fit, rather than an exact match, and x is similar to string x. Moreover, javascript will turn x into a string in an effort to 'help out'
+Console says: Uncaught (x is not defined)
+After defining x as 1, console says: false, my assumption being that variable x is not == string x.
+
 **d)** `x == (x+'')`
 
+True - Because '' is considered an empty string and thus nothing, leaving x to be evaluated to itself.
+Console says: True (x defined as 1, otherwise says undefined)
+
 **e)** `'' == ' '`
 
+False - In this case '' is an empty string, whereas ' ' actually contains content, in this case a space.
+Console says: False
+
+
 **f)** `x = true`
 
+True - Because you are declaring x as true.
+Console says: True
+
 **g)** `var x; x == 'undefined'`
 
+False - In the first step a variable, x, has been defined.
+Console says: false (because of why I say, I'm not so sure)
+
 **h)** `'9'<'10'`
 
+True - javascript will look at these strings as numbers for the operator
+Console says: false. Because in retrospect, javascript looks at the first digit in the string, rather than the string as a whole. Essentially this turns into 9<1, which is false.
+
 **i)** `typeof x + 1 === "number"`
 
+If x has been defined earlier as a number, this will output True.
+Console says: False. After tinkering around with what typeof x + 1 is on its own (after declaring x = 1), it retrieved "number1". So the console is actually adding a 1 to the type of x, which of course is not ultimately === "number"
+
+
 **j)** `typeof x % 2 === "number"`
 
+Similarly, if x has been defined earlier as a number, this will output True.
+Console says: False. I assume for the same reasons that I was wrong on the last example. I think the next example should still be true, however, because it is evaluating the typeof everything in the ()'s rather than the first thing it encounters.
+
+
 **k)** `typeof (x % 2) === "number"`
 
+Again, the same as i and j.
+Console says: True (though not by being the same as i and j!)
+
 **l)** `x++ == ++x`
 
+True, because though x (assuming it is a number) will output a different number on the console if these are placed in individually, they both store the same value in javascript. It is more a matter that x++ reflects the value of x and then adds one afterwards, while ++x adds one and then reflects its new value.
+Console says: False. I believe false because the order of operations is different and so the are not precisely equivalent, though I would have expected true with just double ='s. Perhaps my next prediction should have been false as well then.
+
 **m)** `++x == x++`
```

Comment:
1l & 1m) So, imagine that `var x = 4;`. When `x++ == ++x` goes, the `x++` will return 4, then add one to `x`, making it 5. When the `++x` goes, it will look at x, which is now 5, and add 1 to it, then return 6, so we get `4 == 6` which is clearly false. With `++x == x++`, the `++x` will add one to x and return that (5), and the `x++` will return 5, then add one to x, giving `5 == 5`.